app/model/entity/Assignment.php

```diff
@@ -357,7 +357,9 @@ function ($key, RuntimeEnvironment $env) {
       "maxPointsBeforeFirstDeadline" => $this->maxPointsBeforeFirstDeadline,
       "maxPointsBeforeSecondDeadline" => $this->maxPointsBeforeSecondDeadline,
       "submissionsCountLimit" => $this->submissionsCountLimit,
-      "runtimeEnvironmentsIds" => $this->getRuntimeEnvironmentsIds(),
```

Comment:
just a thought, what about adding new item (`exerciseRuntimeEnvironmentIds`) and keep the old one?